src/main/java/com/example/SOPSbackend/model/converter/VaccinationSlotStatus.java

```diff
@@ -0,0 +1,27 @@
+package com.example.SOPSbackend.model.converter;
+
+import com.example.SOPSbackend.model.VaccinationEntity;
+import com.example.SOPSbackend.model.VaccinationSlotEntity;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import java.time.Instant;
+import java.time.LocalDateTime;
+
+@Getter
+@Setter
+@NoArgsConstructor
+public class VaccinationSlotStatus {
+    public VaccinationSlotStatus(VaccinationSlotEntity vse, VaccinationEntity vaccination)
```

Comment:
No wiadomo, że szczegół, ale nie ma co oszczędzać na nazwa zmiennych. Tutaj jeszcze dwulinijkowa funkcja, więc akceptowalne imo, ale wiadomo o co chodzi.

Reply:
Tak, czasami dobrze jest wykorzystać nazwę parametru żeby przekazać jakąś dodatkową informację ponad to co mówi jego typ. Tutaj akurat ciężko o taką możliwość.